<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0.512, y1:0, x2:0.546, y2:1, stop:0 rgba(21, 35, 225, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>891</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Point Dose Calculation for Brachytherapy&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;with TG-43&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>621</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hi&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;This application was created to perform a independent verification&lt;br/&gt;of the dose calculation for the brachytherapy treatments with a Ir-192&lt;br/&gt;source of Elekta MicroSource&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;Please select the method to input the calculation points:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>570</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color:rgb(240, 240, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:rgb(79, 155, 255)</string>
    </property>
    <property name="text">
     <string>Dicom File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>570</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color:rgb(240, 240, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:rgb(79, 155, 255)</string>
    </property>
    <property name="text">
     <string>Manual Input</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
